bootstrap.py

```diff
@@ -766,6 +776,48 @@ class MEOptions:
     disable_rhn_plugin()
 
 
+def validate_login():
+    """
+    Function to authenticate a user to validate login credentials before
+    running any other API calls. Since we don't have API_PORT setup yet
+    we need to make a test. Strip down output so it doesn't print a
+    big exception and scare users. This function is going to be used
+    before we have the certificates from katello-consumer rpm installed,
+    so we need to disable SSL verification since it's now enabled
+    per default in RHEL7.4+.
+    """
+    port = guess_api_port(options.foreman_fqdn)
```

Comment:
instead of doing this here as a special case, why not using `API_PORT = guess_api_port()` in https://github.com/bergsjoh/katello-client-bootstrap/blob/d1a85da6d7cc9fc4c78bf56c1e325912a7cbca46/bootstrap.py#L851?

and then that should probably be moved *after* the argument parsing, as otherwise `foreman_fqdn` won't be defined yet.